nicoRDL/dotTheory-JS
# JS-106 (JS-350) JavaScript Objects: Loop Through Objects

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-350

## Description

- completed Loop Through Objects tutorial
- completed all sample problems and code challenge
- completed quiz

## Steps to Validate
Code Challenge:
1. Open the index.html in the codeChallenge folder
2. Launch the index file
3. The browser should immediately show a short webpage of pets
4. Return to code; review the script.js file
5. The file should be a valid declaration of an array of Objects and rendered correctly using JS

## Risks
N/A
